config_alerts_notify.go

```diff
@@ -24,15 +26,18 @@ func (c *NotifyConfig) Get() error {
 
 func (c *NotifyConfig) Set() error {
 	_, err := db.Upsert(ConfigCollection, byId(c.SectionId()), bson.M{
-		"$set": bson.M{
-			"smtp": c.SMTP,
-		},
+		"$set": c,
 	})
 	return errors.Wrapf(err, "error updating section %s", c.SectionId())
 }
 
 func (c *NotifyConfig) ValidateAndDefault() error {
-
+	if c.NotificationsPeriodInSecs == 0 {
+		c.NotificationsPeriodInSecs = 60
+	}
+	if c.NotificationsTarget == 0 {
+		c.NotificationsTarget = 20
+	}
```

Comment:
probably <= 0, 

also maybe cap it?

Reply:
Capped at 100 jobs/sec, although i sincerely doubt we'll ever approach that.